app/controllers/collections_controller.rb

```diff
@@ -25,11 +25,23 @@ def load_collection_from_id
 
   def index
     if params[:work_id] && (@work = Work.find_by(id: params[:work_id]))
-      @collections = @work.approved_collections.by_title.includes(:parent, :moderators, :children, :collection_preference, owners: [:user]).paginate(page: params[:page])
+      @collections = @work.approved_collections
+        .by_title
+        .includes(:parent, :moderators, :children, :collection_preference, owners: [:user])
```

Comment:
Considering that all three includes are the same here, maybe a `for_blurb` scope would be nice?